ui/components-react/src/components-react/newEditors/editors/NumericEditor.tsx

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import { Input } from "@itwin/itwinui-react";
+import type { EditorProps } from "../Types.js";
+import type { ValueMetadata } from "../values/Metadata.js";
+import type { NumericValue } from "../values/Values.js";
+
+/**
+ * Numeric value editor that renders `Input` components for changing value.
+ * @internal
+ */
+export function NumericEditor({
+  value,
+  onChange,
+  onFinish,
+  size,
+  disabled,
+}: EditorProps<ValueMetadata, NumericValue>) {
+  const currentValue = getNumericValue(value);
+  return (
+    <Input
+      value={currentValue.displayValue}
+      onChange={(e) =>
+        onChange({
+          rawValue: parseFloat(e.target.value),
+          displayValue: e.target.value,
+        })
```

Comment:
Should this editor have some default rounding? Maybe rounding precision could be specified through metadata?

Reply:
NVM, I guess formatting in the `onChange` callback doesn't make much sense.